src/applications/vaos/tests/mocks/setup.js

```diff
@@ -167,3 +182,64 @@ export async function setPreferredDate(store, preferredDate) {
 
   return router.push.firstCall.args[0];
 }
+
+export function setupExpressCareMocks({
```

Comment:
I think I would put this in mocks/helpers. I've been trying to put service call mocks in there and code that does UI related set up in here. We might need better names/separation for these files.